Assets/Mirror/Editor/Weaver/Extensions.cs

```diff
@@ -137,5 +137,15 @@ public static MethodReference MakeHostInstanceGeneric(this MethodReference self,
             return Weaver.CurrentAssembly.MainModule.ImportReference(reference);
         }
 
+        public static CustomAttribute GetCustomAttribute(this MethodDefinition method, string attributeName)
+        {
+            foreach (CustomAttribute ca in method.CustomAttributes)
+            {
+                if (ca.AttributeType.FullName == attributeName)
+                    return ca;
+            }
+            return null;
+        }
```

Comment:
@paulpach what happens if we have e.g. WriteColor(Color c) in there, and someone adds a WriteColour(Color c) function? would they get notified that an extension for that type already exists?

Reply:
Their method would take precedence.  This is done on purpose so that people can override Mirror's built in serializers,  say if they want compression for Quaternions or something like that.

But I could issue an error instead.

Reply:
makes sense the way it is, I agree